.github/issue_template.md

```diff
@@ -43,4 +44,7 @@ What was the behavior you expected from the library?
 
 What was the behavior you actually observed?
 
-### Anything else
+### Anything else you would like us to know?
+
+Include here information about your environment that is not captured above.
+Or any other information you think might be relevant.
```

Comment:
This is not a complete sentence; please attach it to the previous line by replacing period with a comma.